lib/perl/Genome/Qc/Config.pm

```diff
@@ -45,6 +45,12 @@ sub get_commands_for_alignment_result {
                 use_version => 1.123,
             },
         },
+        samtool_flagstat => {
```

Comment:
Yes, it's a typo (although this hash key doesn't have to match the tool; it can really be anything). It's now fixed.